config/kubernetes-sigs/sig-cli/teams.yaml

```diff
@@ -33,46 +33,35 @@ teams:
     description: admin access to krew
     members:
     - ahmetb
-    - juanvallejo
+    - corneliusweig
     - seans3
     - soltysh
+    - pwittrock
     privacy: closed
   krew-maintainers:
     description: write access to krew
     members:
-    - adohe
     - ahmetb
     - corneliusweig
-    - fabianofranz
-    - janetkuo
-    - liggitt
-    - mengqiy
-    - monopole
-    - pwittrock
```

Comment:
What implications does it have when Phil is member of the admin group, but not in maintainers?

Reply:
honestly not sure, so adding back there.

Reply:
For reference: [GitHub repo permissions comparison](https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization#repository-access-for-each-permission-level).
We setup maintainers to have `write` permissions and admins to have `admin`.
If a person is an admin, they already have all the permissions of someone with `write` access.